support/mender-inventory-update-modules

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+#
+# Returns an the update-modules currently installed on the system
+# For more information on update-modules see:
+# https://hub.mender.io/c/update-modules/13
+#
+
+set -e
+
+FOUND=0
+
+if [ ! -d /usr/share/mender/modules/ ]; then
+  echo update_module'(s)'=""
+  exit 0
+fi
+
+for file in /usr/share/mender/modules/*/*; do
+  if [ ! -x "${file}" ]; then
+    continue
+  fi
+  echo update_module'(s)'=$(basename "${file}")
```

Comment:
I don't get it: won't this override the same `update_module(s)` key for every module? I will produce:

```
update_module(s)=deb
update_module(s)=directory
update_module(s)=docker
update_module(s)=rpm
update_module(s)=script
update_module(s)=single-file
```

Reply:
Yes, but according to our docs:
https://docs.mender.io/client-installation/inventory#lists-of-values
this is correct for a list of values I think.

Reply:
Thanks. Shame on me.

Reply:
https://giphy.com/gifs/vX9WcCiWwUF7G/html5

Feature request for the `mender-test-bot`

Reply:
And Github appareantly